### DISCUSSION TITLE
Deleted characteristic AID =11 IID...

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#539] [Q&A]
Hello,

The program works well (as bridge) but with L1 to debug:
Deleted characteristic AID =12 IID=11671
Deleted characteristic AID =12 IID=11672
Deleted characteristic AID =12 IID=11673
...

I deleted the accessory 12 and get the same line with AID=11

What does it mean ?

**Comment by HomeSpan** [comment]
Sorry, I do not understand this question or what it is in reference to.
